.gitignore

```diff
@@ -38,3 +38,10 @@ prboom2/release
 /prboom2/VisualC2015/ReleaseServer/prboom_server.tlog
 /prboom2/VisualC2015/ReleaseServer
 /build*
+CMakeFiles
+CMakeCache.txt
+Makefile
+*.cmake
+config.h
+prboom-plus.wad
+*.exe
```

Comment:
Please use out-of-tree builds. The `build*` directories filter was added especially for this.

Reply:
https://github.com/coelckers/prboom-plus/pull/37/commits/780c682cc2de4056a71816f4958cfe78dec47ff5#diff-a084b794bc0759e7a6b77810e01874f2L41